ajax/libs/stickyNavbar.js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "stickyNavbar.js",
+  "filename": "jquery.stickyNavbar.min.js",
+  "version": "1.3.4",
+  "description": "Fancy sticky navigation jQuery plugin with smart anchor links highlighting",
+  "dependencies": {
+    "jquery": "^1.10.2"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jbutko/stickyNavbar.js.git"
+  },
+  "keywords": [
+    "stickyNavbar",
+    "navigation",
+    "sticky",
+    "nav",
+    "jQuery"
+  ],
+  "author": "Jozef Butko",
+  "license": "MIT",
+  "homepage": "https://www.jozefbutko.com/stickynavbar/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jbutko/stickyNavbar.js.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.stickyNavbar*.!(css)"
```

Comment:
I excluded it because author just removed `jquery.stickyNavbar.css` in the latest version and the css file is just for demo. Discussed in https://github.com/jbutko/stickyNavbar.js/issues/50.
Should the rule be `"jquery.stickyNavbar*"` and manually add old versions without css file?